src/components/Footer.js

```diff
@@ -0,0 +1,12 @@
+import { Box, Typography } from "@mui/material";
+
+export const Footer = () => {
+  return (
+    <Box
+      component={"footer"}
+      sx={{ background: "white", padding: 1, marginTop: "auto" }}
+    >
+      <Typography variant="body1">Copyright 2023</Typography>
```

Comment:
Better to generate current year, so you don't have to change it manually in few months

Reply:
Yes, I already found how to do it and implemented it.